Makefile

```diff
@@ -1,27 +1,30 @@
+PROFILE=release
+
 .PHONY: build release install uninstall clean test doc reindent
 
 build:
-	jbuilder build @install --dev
+	dune build @install
 
 release:
-	jbuilder build @install
+	dune build @install --profile=$(PROFILE)
 
 install:
-	jbuilder install
+	dune install
```

Comment:
does this need `--profile=$(PROFILE)`?

Reply:
it's not very clear to me; I did test it for a previous package, i.e. I ran `make build` which builds the dev profile, and then ran the install target with release profile; where one might have expected a complaint, the installation went through

Reply:
Looks like this is driven by the .install file created during build, so although dune install accepts a profile it wouldn't actually build anything, so shouldn't be affected by compiler flags from a profile.